system76driver/userdaemon.py

```diff
@@ -139,6 +139,10 @@ def __init__(self, model, rootdir='/'):
         if self.model == "thelio-mira-b1":
             self.mic_dev = 3
             self.line_in_dev = 2
+        elif subprocess.call(["lsusb", "-d", "0414:a00d"]) != "":
```

Comment:
@leviport I think this does need to be 0 instead of ""

Reply:
Hmm, `lsusb -d 0414:a00d` was giving me a long string on the rev1.2 board and no output on other machines. Should I be expecting a `0` instead?

Reply:
Yes. Try running this in python3 interactive shell:

```
import subprocess
subprocess.call(["lsusb", "-d", "0414:a00d"])
```

You will see it return 0 on the rev 1.2 board and 1 on any other system.

Reply:
Yep, you're right. I misunderstood what `subprocess.call()` would do.